### ISSUE TITLE
Textgroups: import and string refresh problems

### ISSUE THREAD

**Issue by indigoxela** [backdrop-contrib/i18n#87]
Textgroups again...

They got removed from core, so handling them has been moved to i18n. That works for the export, but implementing textgroups in the po file input would lead to huge code duplication (take over all po file handling).

The question is now: do the imported strings actually work? They have the right context, but don't have the right textgroup.

**Comment by yorkshire-pudding** [comment]
I'm not sure if this is related at all, but on a D7 upgrade site with `i18n` installed, I got this warning when enabling `token_block` using `bee`:

>  ⚠️  Warning: Declaration of i18n_string_textgroup_cached::string_update($i18nstring, $options = Array) should be compatible with i18n_string_textgroup_default::string_update($i18nstring, array $options = Array)
in require_once() (line 4161 of /app/docroot/core/includes/bootstrap.inc).

Didn't stop anything working and not sure why it would appear on enabling `token_block` but this is the only mention I found of `textgroup` in the issues.

**Comment by indigoxela** [comment]
> I'm not sure if this is related at all, but on a D7 upgrade site with i18n installed, I got this warning...

No, that's unrelated. The base class' method has a type hint for the param and the extending class has not. That's not even really related to textgroups at all, probably rather has to do with the version of PHP the upgrade runs on.

Can you open a new issue for that @yorkshire-pudding ? 🙏

**Comment by yorkshire-pudding** [comment]
Thanks for clarifying @indigoxela . I've opened a separate issue at #145
